queries.sql

```diff
@@ -20,4 +20,12 @@ SELECT MAX(weight_kg) FROM animals WHERE species IN ('digimon');
 SELECT MIN(weight_kg) FROM animals WHERE species IN ('pokemon');
 SELECT MAX(weight_kg) FROM animals WHERE species IN ('pokemon');
 SELECT id, name, date_of_birth, SUM(escape_attempt), COUNT(escape_attempt), SUM(escape_attempt) / COUNT(escape_attempt) as average FROM animals WHERE species IN ('pokemon') AND date_of_birth >= '12/05/1990'::date AND date_of_birth <= '01/01/2021'::date GROUP BY id, name, date_of_birth;
-SELECT id, name, date_of_birth, SUM(escape_attempt), COUNT(escape_attempt), SUM(escape_attempt) / COUNT(escape_attempt) as average FROM animals WHERE species IN ('digimon') AND date_of_birth >= '12/05/1990'::date AND date_of_birth <= '01/01/2021'::date GROUP BY id, name, date_of_birth;
\ No newline at end of file
+SELECT id, name, date_of_birth, SUM(escape_attempt), COUNT(escape_attempt), SUM(escape_attempt) / COUNT(escape_attempt) as average FROM animals WHERE species IN ('digimon') AND date_of_birth >= '12/05/1990'::date AND date_of_birth <= '01/01/2021'::date GROUP BY id, name, date_of_birth;
+
```

Comment:
- [ ] It would be better to perform an update operation for the `animals` table
```
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```